src/app/global-vars.service.ts

```diff
@@ -278,6 +283,11 @@ export class GlobalVarsService {
 
     this.loggedInUser = user;
 
+    // If Jumio callback hasn't returned yet, we need to poll to update the user metadata.
+    if (user.JumioFinishedTime > 0 && !user.JumioReturned) {
+      this.pollLoggedInUserForJumio(user.PublicKeyBase58Check);
```

Comment:
You need to clear the interval before starting a new one. Otherwise switching accounts back and forth will create tons of intervals

Reply:
this interval now gets cleared in the pollLoggedInUserForJumio function